ci_framework/tests/integration/targets/make/files/get_makefiles_env/makefiles/Makefile

```diff
@@ -180,7 +180,7 @@ CINDER                 ?= config/samples/cinder_v1beta1_cinder.yaml
 CINDER_CR              ?= ${OPERATOR_BASE_DIR}/cinder-operator/${CINDER}
 # TODO: Image customizations for all Cinder services
 CINDER_KUTTL_CONF      ?= ${OPERATOR_BASE_DIR}/cinder-operator/kuttl-test.yaml
-CINDER_KUTTL_DIR       ?= ${OPERATOR_BASE_DIR}/cinder-operator/tests/kuttl/tests
+CINDER_KUTTL_DIR       ?= ${OPERATOR_BASE_DIR}/cinder-operator/tests/kuttl/test
```

Comment:
same comment as i just left on ci_framework/tests/integration/targets/make/files/get_makefiles_env/expected_variables_values.yml (tests vs test)

Reply:
I am removing this change - this seemed to have no effect on the cinder_kuttl tests. Will focus on cinder in a separate PR